packages/hub/routes/email-card-drop-requests.ts

```diff
@@ -60,6 +68,65 @@ export default class EmailCardDropRequestsRoute {
     ctx.type = 'application/vnd.api+json';
     return;
   }
+
+  async post(ctx: Koa.Context) {
+    if (!ensureLoggedIn(ctx)) {
+      return;
+    }
+
+    let email = ctx.request.body.data.attributes.email;
+
+    let hash = crypto.createHash('sha256');
+    hash.update(email);
+    let emailHash = hash.digest('hex');
+
+    let verificationCode = (crypto.randomInt(1000000) + '').padStart(6, '0');
+
+    if (isEmail(email)) {
+      const emailCardDropRequest: EmailCardDropRequest = {
+        id: shortUuid.uuid(),
```

Comment:
Why `shortUuuid.uuid()` over `crypto.randomUUID()`?

Reply:
copypaste? 😆 but sure I can replace that

Reply:
oh also it’s [only in Node 16](https://developer.mozilla.org/en-US/docs/Web/API/Crypto/randomUUID#browser_compatibility)

Reply:
It's in node 14 too, the note is misleading

Reply:
hmm strange, autocomplete didn’t show it existing. Oh well!!